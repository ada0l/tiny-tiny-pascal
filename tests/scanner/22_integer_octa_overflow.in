&17777777777 // 2147483647
&20000000000 // 2147483648
