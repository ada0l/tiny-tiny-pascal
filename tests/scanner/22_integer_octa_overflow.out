1	1	LITERAL_INTEGER	2147483647	&17777777777
Exception: Line: 2; Column: 1; Integer overflow